ftw/contacts/simplelayout/resources/ftw.contacts.simplelayout.css

```diff
@@ -0,0 +1,9 @@
+#plone-contentmenu-factories .contenttype-ftw-contacts-member {
+    display: none !important;
+}
+.contenttreeWindowBlocker {
```

Comment:
I'd merge this CSS file into the standard CSS file, as there is not really a CSS dependency to simplelayout.
I'd prefer having only one CSS file.
Maybe you need to make those selectors more specific though :wink: